normandy/recipes/filters.py

```diff
@@ -353,6 +354,35 @@ def capabilities(self):
         return {"jexl.context.env.version", "jexl.transform.versionCompare"}
 
 
+class ProfileCreateDateFilter(BaseFilter):
+
+    type = "profile_creation_date"
+    direction = serializers.CharField()
+    date = serializers.DateField()
+
+    def to_jexl(self):
+        direction = self.initial_data["direction"]
+        date = self.initial_data["date"]
+
+        days = (datetime.strptime(date, "%Y-%M-%d") - datetime(1970, 1, 1)).days
+
+        if direction == "olderThan":
+            symbol = "<"
+        else:
+            symbol = ">"
```

Comment:
I'm a little worried about this logic. It would be pretty easy to have a mistake like `direction: "older-than"` which would surprisingly result in a filter that targets newer users (since it doesn't match exactly `olderThan`).

What would you think about checking for exactly `olderThan` or `newerThan`, and throwing an error if that's not the case?

Reply:
That makes sense.

Reply:
is validation error the right error to be throwing? I see that error being thrown in other filters.

Reply:
see code right below this comment...